lib/elixir_boilerplate/application.ex

```diff
@@ -8,12 +8,7 @@ defmodule ElixirBoilerplate.Application do
   alias ElixirBoilerplateWeb.Endpoint
 
   def start(_type, _args) do
-    import Supervisor.Spec
-
-    children = [
-      supervisor(Repo, []),
-      supervisor(Endpoint, [])
-    ]
+    children = [Repo, Endpoint]
```

Comment:
We could even remove the aliases:

```suggestion
    children = [
      ElixirBoilerplate.Repo,
      ElixirBoilerplateWeb.Endpoint
    ]
```

Reply:
Yep, much cleaner! 👌